SDCTA/src/components/modal/styleObjects/signupStyle.tsx

```diff
@@ -0,0 +1,86 @@
+export const style = {
+    //location styling
+    top: '50%',
+    left: '50%',
+    transform: 'translate(-50%, -50%)',
+    width: '625px',
+    height: '423px',
+    //box formatting
```

Comment:
I'd personally put a blank line above the comment in order to differentiate the sections better but just preference for readability